scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -73,6 +74,11 @@ def unpack_ttest_result(res):
 inaccuracy_messages = {"Precision loss occurred in moment calculation",
                        "Sample size too small for normal approximation."}
 
+# For some functions, nan_policy='propagate' should not just return NaNs
+override_propagate_funcs = {stats.mode}
+
+# For some functions, empty input should not result in all-NaN output
+override_empty_funcs = {stats.mode}
```

Comment:
This is unused. Is this like a note that `stats.mode` is different?

Reply:
Oops. I think that was from before a refactoring. Removed.